docs/streams/index.html

```diff
@@ -201,7 +201,23 @@ <h3 style="max-width: 75rem;">The easiest way to write mission-critical real-tim
             </pre>
         </div>
     </div>
-
+     <h3 class="customer-title">See how Kafka Streams is being used</h3>
+    <div class="customer__cards">
+        <div class="customer__card">
+          <div class="customer__card__icon">
+            <img src="/0110/images/icons/rabobank.png">
+          </div>  
+            <span class="customer__card__label">Rabobank is one of the 3 largest banks in the Netherlands. Its digital nervous system, the Business Event Bus, is powered by Apache Kafka and Kafka Streams.
+           </span>
+           <a href="https://www.confluent.io/blog/real-time-financial-alerts-rabobank-apache-kafkas-streams-api/">Learn More</a>
+        </div>
+        <div class="customer__card customer-right" >
+          <div class="customer__card__icon" >
+            <img src="/0110/images/icons/zalando.png">
+          </div>  
+            <span class="customer__card__label">As the leading online fashion retailer in Europe, Zalando uses Kafka as an ESB (Enterprise Service Bus), which helps us in transitioning from a monolithic to a micro services architecture. Using Kafka for processing event streams enables our technical team to do near-real time business intelligence.</span>
+            <a href="https://kafka-summit.org/sessions/using-kstreams-ktables-calculate-real-time-domain-rankings/">Learn More</a>
+        </div>
```

Comment:
I think you're missing a `</div>` to close up the `.customer__cards` div that starts this. This is causing the following pagination div to be included in this one and it is not rendering correctly. 
<img width="966" alt="screen shot 2017-09-19 at 9 08 32 pm" src="https://user-images.githubusercontent.com/47366/30626443-c55a1368-9d7e-11e7-8968-deb65093d026.png">

Keeping things indented clearly tends to make these things obvious -- might help to use an editor that will help with the indenting and that you can configure for the 4-space indent used in the rest of these files (generally, I don't think we've perfectly enforced this). Then it's easy to notice if it doesn't think code after your additions is at the right level already -- a mismatching tag usually becomes pretty easy to spot.